src/code42cli/cmds/securitydata.py

```diff
@@ -97,7 +98,25 @@
 file_category_option = click.option(
     "--file-category",
     multiple=True,
-    type=click.Choice(list(FileCategory.choices())),
+    type=MapChoice(
+        choices=list(FileCategory.choices()),
+        extras_map={
+            "AUDIO": "Audio",
+            "DOCUMENT": "Document",
+            "EXECUTABLE": "Executable",
+            "IMAGE": "Image",
+            "PDF": "Pdf",
+            "PRESENTATION": "Presentation",
+            "SCRIPT": "Script",
+            "SOURCE_CODE": "SourceCode",
+            "SPREADSHEET": "Spreadsheet",
+            "VIDEO": "Video",
+            "VIRTUAL_DISK_IMAGE": "VirtualDiskImage",
+            "ARCHIVE": "Archive",
+            "ZIP": "Archive",
+            "Zip": "Archive",
```

Comment:
These didn't exist in the CLI, previously but since the Console text is "Zip", and py42's attribute value is ".ZIP" I figured it can't hurt to let those just work.